.analysis_options

```diff
@@ -1,2 +1,29 @@
 analyzer:
   strong-mode: true
+linter:
+  rules:
```

Comment:
This is completely irrelevant.
The library does not need these rules, so it's meaningless boilerplate. Just drop it.